compiler/rustc_codegen_cranelift/src/compiler_builtins.rs

```diff
@@ -43,7 +43,7 @@ builtin_functions! {
     fn __divti3(n: i128, d: i128) -> i128;
     fn __umodti3(n: u128, d: u128) -> u128;
     fn __modti3(n: i128, d: i128) -> i128;
-    fn __rust_u128_mulo(a: u128, b: u128) -> (u128, bool);
+    fn __rust_u128_mulo(a: u128, b: u128, oflow: &mut bool) -> u128;
```

Comment:
In the compiler-builtins PR you are using `&mut i32` instead of `&mut bool`.

Reply:
Thanks. I had `bool` originally but changed to `i32` for consistency with the C APIs.